src/tests/excalidrawPackage.test.tsx

```diff
@@ -224,4 +224,22 @@ describe("<Excalidraw/>", () => {
       });
     });
   });
+
+  describe("Test autofocus prop", () => {
+    it("should not focus when autofocus is false", async () => {
+      const { container } = await render(<Excalidraw autofocus={false} />);
```

Comment:
```suggestion
      const { container } = await render(<Excalidraw />);
```
let's skip passing here since its false by default and that way default prop will be tested as well